### PR TITLE
Added OpenEye error to quacpac failure

### PR DESCRIPTION
**Pull request by dotsdl**
Addresses #379.

We now include the OpenEye `quacpac` error in our `Exception` message if charges cannot be assigned.

To observe an example of this failure, we can try running AM1-BCC on a Lithium ion (credit @j-wags):
```python
from openforcefield.topology import Molecule
from openforcefield.topology import Topology
from openforcefield.typing.engines.smirnoff import ForceField

molecule = Molecule.from_smiles('[Li+1]')
forcefield = ForceField('test_forcefields/smirnoff99Frosst.offxml')

topology = Topology.from_molecules(molecule)
system = forcefield.create_openmm_system(topology)
```
This yields:
```
Exception: Unable to assign charges; OE error: "Warning: AM1BCC model does not support molecules containing Li
Warning: OEBCCPartialCharges: Unable to assign all BCCs in mol 
Warning: OESEmpBCCPartialCharges: Stopping the charging process at the first stage for mol 
"
```

- [x] Tag issue being addressed
- [x] Add [tests](https://github.com/openforcefield/openforcefield/tree/master/openforcefield/tests)
- [x] Update docstrings/[documentation](https://github.com/openforcefield/openforcefield/tree/master/docs), if applicable
- [x] Update [changelog](https://github.com/openforcefield/openforcefield/blob/master/docs/releasehistory.rst)

### CODE REVIEW

**Comment by dotsdl** [comment]
I don't believe there are any docs to update for this one; this is ready for review.

**Comment by mattwthompson** [comment]
Not sure why [this](https://pipelines.actions.githubusercontent.com/u2Wf9sTwFK9NZj8rRqcrkPO1UF3tPeGIdu4NysSHujaRyWbUAY/_apis/pipelines/1/runs/78/signedlogcontent/3?urlExpires=2020-04-10T18%3A39%3A37.1152224Z&urlSigningMethod=HMACV1&urlSignature=mjjzlJ8f4j14Jwkj%2FeOZ8mm5yQw9ct3Z9zQB%2F97bUvU%3D) test failed (only once in the array):
```
2020-04-10T02:27:27.2919470Z examples/check_dataset_parameter_coverage/check_parameter_coverage::ipynb::Cell 6 FAILED [  0%]

**Comment by dotsdl** [comment]
Thanks @mattwthompson!

Looks like we're getting failures with checking links like this: https://dx.doi.org/10.1021/acs.jctc.8b00640.

Is this necessary in our tests? The JCTC site has a DDOS checker and it may not be liking all the requests its getting around the same time from Github Actions.

**Comment by mattwthompson** [comment]
I figure it's necessary to mark those that we know have DDOS protection as xfail (my preference), find a workaround, or drop them altogether

File: openforcefield/tests/test_toolkits.py [hunk 1]
```diff
@@ -534,6 +534,21 @@ def test_compute_partial_charges_net_charge(self):
             charge_sum += pc
         assert 0.999 * unit.elementary_charge < charge_sum < 1.001 * unit.elementary_charge
 
+    @pytest.mark.skipif(not OpenEyeToolkitWrapper.is_available(), reason='OpenEye Toolkit not available')
+    def test_compute_partial_charges_failure(self):
+        """Test OpenEyeToolkitWrapper compute_partial_charges() on a molecule it cannot assign charges to"""
+
+        toolkit_wrapper = OpenEyeToolkitWrapper()
+        smiles = '[Li]'
```
Comment (j-wags) [inline openforcefield/tests/test_toolkits.py:542]:
```suggestion
        smiles = '[Li+]'
```

**Comment by dotsdl** [comment]
Thanks everyone! Merged!

**Pull request merged**